channels/email/settings.rst

```diff
@@ -22,14 +22,36 @@ Notification Sender: Default value ``Notification Master <noreply@#{config.fqdn}
    This address is relevant for agent notifications and password reset mails
    (also affects customers).
 
-Additional follow-up detection
-   In some situations the normal follow-up detection is not enough.
-   This might be due to missing references in the subject
-   (the ticket hook and number). These options can help to recognize follow-ups
-   to existing tickets.
-
-   Please note that searching in attachment and body might lead to *false
-   follow-up detection*.
+Additional follow-up detection: Default value ``Subject & References``
+   Adjust how Zammad checks if an incoming email is a follow-up or not.
+   If you disable all checks, Zammad only checks if a matching ticket hook &
+   number is present in the subject.
+
+   Subject & References
+      Checks subject (ticket hook & number) and message IDs of emails. If
+      one of them matches with existing ones, an email is considered as a
+      follow-up.
+
+   References
+      This can be used as an alternative to "Subject & References" if
+      you don't want to have the subject of the email checked but only message
+      IDs.
```

Comment:
I think we should reword this, to mention explicitly that check will only match existing articles in the system with the same references (message ID).